logic/src/commonMain/kotlin/com/wire/kalium/logic/data/user/UserMapper.kt

```diff
@@ -71,7 +75,8 @@ internal class UserMapperImpl(private val idMapper: IdMapper = MapperProvider.id
             accentId = userProfileDTO.accentId,
             team = userProfileDTO.teamId,
             previewAssetId = userProfileDTO.assets.getPreviewAssetOrNull()?.key,
-            completeAssetId = userProfileDTO.assets.getCompleteAssetOrNull()?.key
+            completeAssetId = userProfileDTO.assets.getCompleteAssetOrNull()?.key,
+            availabilityStatus = null
```

Comment:
What if I have a valid local value of the status? Shouldn't we make sure to update this value correctly (on the UserRepository?) with the old value to avoid nulling the user status every time we try to fetch the latest state of the user remotely?

Reply:
in DB `user_availability_status` change is ignored in user inserting and updating, it can be changed only by `updateUserAvailabilityStatus` method. So that `null`  (after update it will be `NONE`) here wouldn't break anything